Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+#!groovy
+
+node {
+  checkout scm
+
+  def dockerRepoName = 'zooniverse/annotate'
+  def dockerImageName = "${dockerRepoName}:${env.BUILD_ID}"
+  def newImage = null
+
+  stage('Build Docker image') {
+    newImage = docker.build(dockerImageName)
+    newImage.inside {
+      sh 'npm install'
+    }
+  }
+
+  stage('Deploy') {
+    if (BRANCH_NAME == 'staging') {
+      newImage.inside {
+        sh 'export BUCKET="zooniverse-static"; export PREFIX="preview.zooniverse.org/annotate/"; export BASE_URL="https://preview.zooniverse.org/annotate"; npm run build && npm run deploy'
```

Comment:
Aren't these env vars still set via the gulp tasks? Do they have to be specified here

Reply:
I think I left them in - I'll remove them if so. I thought it was simpler to have a single generic deploy command that picks up env vars for config, rather than keeping deploy-specific config in with the code.

Reply:
Cool - OOI does this make it harder to manually deploy if we need to?

Reply:
I shouldn't think so - you'd just run this command locally

Reply:
make a note in the readme pointing to this code perhaps? A new dev that didn't write the Jenkinsfile / doesn't know about them might not understand this.

Reply:
Done